controllers/wizard/fileUpload/form/PKPSubmissionFilesUploadBaseForm.inc.php

```diff
@@ -194,25 +194,32 @@ public function getSubmissionFiles()
                     $this->_submissionFiles = [];
                 } elseif ($reviewRound) {
                     // Retrieve the submission files for the given review round.
-                    $submissionFilesIterator = Services::get('submissionFile')->getMany([
-                        'reviewRoundIds' => [(int) $reviewRound->getId()],
-                        'submissionIds' => [(int) $this->getData('submissionId')],
-                    ]);
+                    $collector = Repo::submissionFiles()
+                        ->getCollector()
+                        ->filterByReviewRoundIds([(int) $reviewRound->getId()])
+                        ->filterBySubmissionIds([(int) $this->getData('submissionId')]);
```

Comment:
I know this was a preexisting problem, but whenever we're filtering by a user-specified submission ID, it's always worth making sure it belongs to the current context (journal) as well.

Reply:
This still needs fixing.